content/docs/user-guide/packaging/project-export/project-export-cli.md

```diff
@@ -84,33 +87,38 @@ To use the export script, first ensure you have the necessary seedlist files for
+| [`--asset-bundling-path`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L345) | Designates where the artifacts from the asset bundling process will be written to before creation of the package. If not specified, default is `<o3de_project_path>/build/asset_bundling`.  | no |
+| [`--max-bundle-size`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L349) | Specify the maximum size of a given asset bundle.  | no |
+| [`--game-launcher or --no-game-launcher`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L352-L357) | This determines whether or not to build the Game Launcher. Which flag shows here depends on export-project-configure defaults for `option.build.game.launcher`. | no |
+| [`--server-launcher or --no-server-launcher`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L359-L364) | This determines whether or not to build the Server Launcher on a platform. Which flag shows here depends on export-project-configure defaults for `option.build.server.launcher`. | no |
+| [`--unified-launcher or --no-unified-launcher`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L367-L372) | This determines whether or not to build the Unified Launcher. Which flag shows here depends on export-project-configure defaults for `option.build.unified.launcher`. | no |
+| [`--platform`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L389) | The expected target platform that assets will be processed and built for. If not specified, the user's host platform will be used. | no |
+| [`--engine-centric or --project-centric`](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/ExportScripts/export_source_built_project.py#L238) | Determines whether to use the engine-centric work flow to export the project. Which flag shows here depends on export-project-configure defaults for `option.engine.centric`. | no |
+| [`--monolithic or --non-monolithic`](https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/ExportScripts/export_source_built_project.py#L381-L386) | Determines whether to use the monolithic builds when exporting the project. Which flag shows here depends on export-project-configure defaults for `option.build.monolithic`. | no |
```

Comment:
I thought this now defaults to true?

Reply:
The user can set the default at any time, due to how `export-project-configure` works. I outline both possibilities here in case they ever change the default so they know what to look for.